homeassistant/components/media_player/plex.py

```diff
@@ -0,0 +1,199 @@
+"""
+homeassistant.components.media_player.plex
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Provides an interface to the Plex API
+
+Configuration:
+
+To use Plex add something like this to your configuration:
+
+media_player:
+  platform: plex
+  name: plex
+  user: plex
+  password: my_secure_password
+
+Variables:
+
+name
+*Required
+The name of the backend device
+
+device
+*Required
+The frontend device (Look under settings > devices).
+
+user
+*Required
+The Plex username
+
+password
+*Required
+The Plex password
+"""
+
+import logging
+
+from homeassistant.components.media_player import (
+    MediaPlayerDevice, SUPPORT_PAUSE, SUPPORT_PREVIOUS_TRACK,
+    SUPPORT_NEXT_TRACK, MEDIA_TYPE_TVSHOW, MEDIA_TYPE_VIDEO)
+from homeassistant.const import (
+    STATE_IDLE, STATE_PLAYING, STATE_PAUSED, STATE_UNKNOWN)
+from plexapi.myplex import MyPlexUser
+
+REQUIREMENTS = ['https://github.com/miniconfig/python-plexapi/archive/'
+                '437e36dca3b7780dc0cb73941d662302c0cd2fa9.zip'
+                '#python-plexapi==1.0.2.5']
```

Comment:
Ah, sorry about that.  I changed the version to 1.0.2.5 because I was concerned about my fork being confused with the "official" python-plexapi which is at 1.0.2 (and I figured the next official version could very well be 1.0.3) - Should I change my fork to be 1.0.2.5, or is my concern unfounded in the first place?